### PR TITLE
Section11,reworked

### PR DESCRIPTION
**Pull request by JoRembert**
section 11 worked over, looks pretty good.

### CODE REVIEW

File: src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/11.md [hunk 1]
```diff
@@ -2,36 +2,36 @@
 ##### Historical period: the 18th century CE
 ##### Scene of Action: Burton upon Trent
 
-In the beginning of the 18th century, London was the world's brewing capital by a vast margin. Hansa hadn't recovered after the Thirty Years' War of 1618-1648; the Netherlands, which was severed by the wars of religion even stronger than the League, at the end of the 17th century had rebuilt its economics, but not the brewing industry — because of multiple reasons, from high taxes to the grape wine and strong alcohol expansion throughout Northern Europe. The growing popularity of porter strengthened London's superiority even further.
+In the beginning of the 18th century, London was the world's brewing capital by a vast margin. Hansa hadn't recovered after the Thirty Years' War of 1618-1648 and the Netherlands was severed by the wars of religion even more so than the League. By the  end of the 17th century it had rebuilt its economics, but not the brewing industry for a host of reasons: high taxes compared to grape wine and strong alcohol expansion throughout Northern Europe. The growing popularity of porter strengthened London's superiority even further.
```
Comment (twirl) [inline src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/11.md]: I don't actually know whether beer taxes were higher than those on wine. They were just high. Changed it to 'high taxes and strong alcohol expansion throughout Northern Europe, to name a few' — please review

File: src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/11.md [hunk 2]
```diff
@@ -2,36 +2,36 @@
...
 
-Burtoners owed this expansion to water: first, the River Trent that gave access to British ports; second, the unique qualities of local water sources. Burton's water was hard and rich in calcium and magnesium sulfates. It turned out that this water suited brewing exceptionally well, stimulating the growth of yeasts and allowing for intensive hopping. Burton beer was more carbonated and much more clear than the London one (and it also got the unique ‘Burton snatch’ — the fleeting ‘aroma’ of sulfur that occurred shortly after pouring).
+Burtoners owed this expansion to water. First, the River Trent gave access to British ports. Second, it held utterly unique local water qualities. Burton's water was hard and rich in calcium and magnesium sulfates. It turned out that this water suited brewing exceptionally well, stimulating the growth of yeasts and allowing for intensive hopping. Burton beer was more carbonated and much clearer than London beer (and it also contained the unique ‘Burton snatch’ — the fleeting ‘aroma’ of sulfur that occurred shortly after pouring).
```
Comment (twirl) [inline src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/11.md:13]: I read it as 'water from Trent river had unique qualities', which is not true: Burtoners were using boreholes AFAIK.

File: src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/11.md [hunk 3]
```diff
@@ -2,36 +2,36 @@
...
+The history of Burton Ale consists of rises and falls, half a century passing between each. The period of prosperity based on the Baltic trade did not last long. In 1783, Russian authorities imposed a 300 percent tax on beer imports. They were then obliged to follow the *Continental System* during the Napoleonic era. The annexation of Poland by Russia  had also closed the Polish market for English brewers. Finally, in 1822, a new Russian customs tariff was introduced that effectively banned various and sundry imports (including beer) from Britain. As a result, at the beginning of the 19th century, the Burton brewing industry deteriorated (four out of 15 breweries were closed, several others sold[ref:brewing-in-burton:29]()) — just to make the rebirth from the ashes (the story of which shall be told in the next chapter) even more spectacular.
```
Comment (twirl) [inline src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/11.md]: I would rephrase it. It was Napoleon who tried to enforce the blockade, and once some territories (including Russian ones) were falling into his control, they migh have joined the blockade. Too complex sentence to explain it, so I would rather leave it simply 'the Napolean-inspired Continental Blockade against British goods' or smth. Also, I would prefer 'blockade' over 'system' on the same grounds, as 'system' will require explanations.

Comment (JoRembert) [reply src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/11.md]: Russia was subdued by Napoleon by 1803, Really the whole continent. The continental system is the historic term, though without a Navy it cut France off from things it needed, like sugarcane. i made the change.

File: src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/11.md (hunk 2 above)
Comment (JoRembert) [reply src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/11.md:13]: yes!
